tools/export/mcuxpresso/__init__.py

```diff
@@ -0,0 +1,758 @@
+"""
+mbed SDK
+Copyright (c) 2011-2016 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Title: MCUXpresso exporter.
+
+Description: Creates a managed build project that can be imported by 
+the MCUXpresso IDE from NXP
+
+Based on GNU ARM Eclipse Exporter from Liviu Ionescu <ilg@livius.net>
+modified for MCUXpresso by Johannes Stratmann <jojos62@online.de>
+"""
+
+import copy
+
+from os.path import splitext, basename, exists
+from random import randint
+
+from tools.export.gnuarmeclipse import GNUARMEclipse
+from tools.export.exporters import apply_supported_whitelist
+from tools.targets import TARGET_MAP
+from tools.utils import NotSupportedException
+from tools.build_api import prepare_toolchain
+
+ 
+# =============================================================================
+
+
+class UID:
```

Comment:
You probably could just import this class from `tools.export.gnuarmeclipse`. Could you do that?